waku-bindings/src/node/mod.rs

```diff
@@ -26,6 +27,7 @@ use crate::general::{
 };
 
 pub use config::{WakuLogLevel, WakuNodeConfig};
+pub use disc_v5::{waku_discv5_start, waku_discv5_stop};
```

Comment:
This is incomplete. We should encode this within the node itself. Similar to other methods that are only available depending on connection conditions.